04 auth

Andruha0013 opened the pull request:
Добрий день, перевірте д/з будь-ласка

Andruha0013 committed:
main functionality rebuild

Andruha0013 committed:
clear unless files

Andruha0013 committed:
all done

Andruha0013 committed:
remove c.logs

Andruha0013 committed:
Update auth.js

Harryk13 commented on controllers/contacts.js:
```diff
@@ -0,0 +1,159 @@
+
+const getContactById = async (req, res, next) => {
+	try {
+		const contact = await Contact.findOne({ _id: req.params.contactId });
```
Усі роути на контакти було б добре обмежити по овнеру. Стосовно цього ви можете почитати в загальному каналі з ментором.

Andruha0013 replied:
Добрий день, переробив "всі контакти", "редагування контакту" та видалення.
Зробив через findOne... фільтр {id,owner}, більш адекватного варіанту не придумам, якщо є такий буду вдячний за підказку.
Дякую.

Andruha0013 committed:
Update contacts controllerr

replaced work with ID to Filter,
now update, remove and getall - works by filter that has 'owner' field
